tools/ci/platforms/darwin.sh

```diff
@@ -103,10 +103,12 @@ bloaty() {
     # https://github.com/google/bloaty/pull/326
     git checkout 52948c107c8f81045e7f9223ec02706b19cfa882
     mkdir -p "${NUTTXTOOLS}"/bloaty
-    cmake -D BLOATY_PREFER_SYSTEM_CAPSTONE=NO -DCMAKE_SYSTEM_PREFIX_PATH="${NUTTXTOOLS}"/bloaty
```

Comment:
Maybe we can try to use latest version for MacOS. I see that PR mentioned in the commit is already merged, so maybe we can pick version from `brew`?

Reply:
@pkarashchenko It is necessary to try. But now I don't have much free time so in a future PR.

Reply:
Sure. Assume that this comment is just a "note"